clippy_lints/src/matches.rs

```diff
@@ -664,6 +686,25 @@ fn check_match_as_ref(cx: &LateContext<'_, '_>, ex: &Expr, arms: &[Arm], expr: &
     }
 }
 
+fn check_pats_wild_match(cx: &LateContext<'_, '_>, _ex: &Expr, arms: &[Arm], _expr: &Expr) {
+    for arm in arms {
+        if let PatKind::Or(ref fields) = arm.pat.kind {
+            // look for multiple fields where one at least matches Wild pattern
+            if fields.len() > 1 && fields.into_iter().any(|pat| is_wild(pat)) {
```

Comment:
You can reduce the `any` call to just `.any(is_wild)`.

Reply:
Right, I always forget about that one